src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1123,7 +1126,20 @@ bool entitiesQuery
 
         /* Attributes part */
 
-        std::vector<BSONElement> queryAttrV = r.getField(ENT_ATTRS).Array();
+        std::vector<BSONElement> queryAttrV;
```

Comment:
BTW, maybe we should apply the strategy of detecting the "$err" after querying in all the places in which we use `connection->query()`. Probably not as part of this PR (except in the case it doesn't involve to much work), but opening and tech debt issue about it.